src/Model/ModelManager.php

```diff
@@ -503,6 +503,13 @@ public function getUrlSafeIdentifier($entity)
 
     public function addIdentifiersToQuery($class, ProxyQueryInterface $query, array $idx)
     {
+        if ([] === $idx) {
```

Comment:
Can you add a phpdoc for nonempty arrays?
https://phpstan.org/writing-php-code/phpdoc-types#general-arrays